utils/jsonconfigwriter.py

```diff
@@ -0,0 +1,66 @@
+import json
+
+'''
+===============================================================================
+CLASS JsonConfigWriter
+===============================================================================
+Intended to create a config JSON file to start the dmprd.
+It uses a given template config.json with default values for every required
+field.
+Set...(): methods are used to modify the default values.
+method writeOutputFile(...): is used to write a new JSON config file with the
+modified values at a given path
+===============================================================================
```

Comment:
This shitty lines are somewhat, yeah, say, shitty. Looks like Windows programmer try to be hip. No just remove these too. Use standard python package documentation. Try look at the standard cpython and how they document their packages and classes.

Reply:
documentation changed in latest commit